src/uwtools/config.py

```diff
@@ -70,6 +72,17 @@ def __init__(self, config_path=None):
 
         self.config_path = config_path
 
+        #initialize logger with parent inheritance
+        #pylint: disable=no-member
+        py_caller = getmodule(stack()[1][0])
+        if py_caller.__name__ in logging.root.manager.loggerDict:
+            _name = f"{py_caller.__name__}.{__name__}"
+        else:
+            _name = __name__
+
+        self.log = logging.getLogger(_name)
+        self.log.debug("Beginning logging with {name}", name=_name)
```

Comment:
Would we not want the test to make sure that this line ends up in the log?

Reply:
Hm. I may have reached the stage in revisions where I'm outsmarting myself....